spec/controllers/admins_controller_spec.rb

```diff
@@ -112,8 +112,8 @@
       let(:organization) { create(:organization) }
       let(:facility_group) { create(:facility_group, organization: organization) }
       let(:facilities) { create_list(:facility, 2, facility_group: facility_group) }
-      let(:manager) { create(:admin, :manager, :with_access, resource: organization) }
-      let(:power_user) { create(:admin, :power_user) }
+      let(:manager) { create(:admin, :manager, :with_access, resource: organization, receive_approval_notifications: true) }
```

Comment:
Let's add a `receive_approval_notifications { true }` to the users factory also.